port_forwarding_in_existing.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright (C) 2020 Private Internet Access, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+if [[ ! $PIA_USER || ! $PIA_PASS || ! $PIA_HOSTNAME || ! $PIA_GATEWAY ]]; then
+  echo 'This script requires 4 env vars:'
+  echo 'PIA_USER            - PIA username'
+  echo 'PIA_PASS            - PIA password'
+  echo 'PIA_HOSTNAME        - name of the server, required for ssl'
+  echo 'PIA_GATEWAY         - the vpn gateway you are connected to.'
+  exit 1
+fi
+
+token="$(PIA_SERVER_META_HOSTNAME=$PIA_HOSTNAME \
+  PIA_SERVER_META_IP=10.0.0.1 \
```

Comment:
Will it work in every region?

Reply:
Yes, according to this [comment](https://github.com/pia-foss/manual-connections/issues/10#issuecomment-706302093). Also it's worked for every region i've tried.

Reply:
@pbruski Yes, according to this [comment](https://github.com/pia-foss/manual-connections/issues/10#issuecomment-706302093). Also it's worked for every region i've tried.